### PR TITLE
Fix https://github.com/brisbanesocialchess/.github/issues/102

### PR DESCRIPTION
**Pull request by BaseMax**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -32,8 +33,8 @@ We're always looking for enthusiastic individuals to join our community! Whether
 
 Connect with other chess enthusiasts, join discussions, and participate in exclusive events by joining our Discord server!
 
-<p align="center">
-    <a href="https://discord.com/invite/JWBKhQmzvD" style="text-decoration:none;">
+<p align=center>
+    <a href="https://discord.com/invite/JWBKhQmzvD" style=text-decoration:none>
```
Comment (jbampton) [inline profile/README.md:37]: Why are the quotes removed ?

Comment (BaseMax) [reply profile/README.md]:
```
profile/README.md:38:11: typography.symbols.curly_quotes Use curly quotes “”, not straight quotes "".
profile/README.md:40:142: typography.symbols.curly_quotes Use curly quotes “”, not straight quotes "".
```

**Comment by BaseMax** [comment]
Now it's ready for you, John.

**Review by jbampton — CHANGES_REQUESTED** [review]
HTML need to be valid with quotes.

So just use a proselint ignore in a config

**Comment by BaseMax** [comment]
> This branch is out-of-date with the base branch
Merge the latest changes from main into this branch.

Update

**Pull request closed without merging**